code/Extension/UserFormFieldEditorExtension.php

```diff
@@ -243,10 +246,7 @@ public function onAfterDuplicate($oldPage, $doWrite, $manyMany)
         foreach ($oldPage->Fields() as $field) {
             /** @var EditableFormField $newField */
             $newField = $field->duplicate(false);
-            $newField->ParentID = $this->owner->ID;
-            $newField->ParentClass = $this->owner->ClassName;
-            $newField->Version = 0;
-            $newField->write();
+            $this->getOwner()->Fields()->add($newField);
```

Comment:
Matches how this is done in `duplicateHasManyRelation` which is what we're effectively doing here.